### PR TITLE
chore: typeScript for type of toast message

### PR DESCRIPTION
**Pull request by seyedmostafahasani**
I added 3 different types ('success' | 'error' | 'info').

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
 
 export type ReactChildren = React.ReactNode;
 
-export type ToastType = string;
+export type ToastType = 'success' | 'error' | 'info';
```
Comment (calintamas) [inline src/types/index.ts:13]:
```suggestion
export type ToastType = 'success' | 'error' | 'info' | (string & {});
```
to allow other strings as well (there can be custom toast types). 

`(string & {})` is [a workaround](https://stackoverflow.com/a/61048124), to keep the type wide, but also provide auto-complete suggestions in the editor

**Comment by calintamas** [comment]
@seyedmostafahasani Thank you! I'll release a new version

**Comment by seyedmostafahasani** [comment]
> @seyedmostafahasani Thank you! I'll release a new version

Thanks Calin! I'll work on other issues.

**Comment by calintamas** [comment]
@seyedmostafahasani Thank you, I appreciate it! I'm here for support, when needed.

**Pull request merged**